spring-boot-project/spring-boot/src/test/java/org/springframework/boot/logging/logback/LogbackLoggingSystemTests.java

```diff
@@ -487,6 +490,43 @@ public void testDateformatPatternProperty() {
 				.containsPattern("\\d{4}-\\d{2}\\-\\d{2}T\\d{2}:\\d{2}:\\d{2}");
 	}
 
+	@Test
+	public void testDefaultJmxConfigurator() throws Exception {
+		this.loggingSystem.beforeInitialize();
+		assertThat(ManagementFactory.getPlatformMBeanServer().queryMBeans(
+				new ObjectName("ch.qos.logback.classic:Name=default,*"), null))
+						.hasSize(1);
+	}
+
+	@Test
+	public void testExplicitJmxConfigurator() throws Exception {
+		this.loggingSystem.beforeInitialize();
+		this.loggingSystem.initialize(this.initializationContext,
+				"classpath:logback-jmxconfigurator.xml", null);
+		assertThat(ManagementFactory.getPlatformMBeanServer().queryMBeans(
+				new ObjectName("ch.qos.logback.classic:Name=default,*"), null))
+						.hasSize(1);
+	}
+
+	@Test
+	public void testShutdownUnregistersJmxConfigurator() throws Exception {
+		this.loggingSystem.beforeInitialize();
+		this.loggingSystem.getShutdownHandler().run();
+		assertThat(ManagementFactory.getPlatformMBeanServer().queryMBeans(
+				new ObjectName("ch.qos.logback.classic:Name=default,*"), null)).isEmpty();
+	}
+
+	@Test
+	public void testTwoLoggingSystemsYieldSingleJmxConfigurator() throws Exception {
```

Comment:
I don't consider this to be a good thing actually. Which logging configuration are you effectively managing with such setup? Is there a way to provide another name than `default` so that we get one MBean per app instead?